pom.xml

```diff
@@ -1063,9 +1063,10 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-release-plugin</artifactId>
-                    <version>2.5.3</version>
+                    <version>3.0.0</version>
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
+                        <preparationProfiles>release</preparationProfiles>
                         <releaseProfiles>release</releaseProfiles>
```

Comment:
This way any Javadoc and GPG issues can already be surfaced during due `release:prepare` phase, instead of the `release:perform` phase.